cert_manager/manifest.json

```diff
@@ -0,0 +1,39 @@
+{
+  "display_name": "cert-manager",
+  "maintainer": "help@datadoghq.com",
+  "manifest_version": "1.0.0",
+  "name": "cert_manager",
+  "metric_prefix": "cert_manager.",
+  "metric_to_check": "cert_manager.prometheus.health",
+  "creates_events": false,
+  "short_description": "Track all your cert-manager metrics with Datadog",
+  "guid": "17888d72-430b-4c0b-b557-e08fd96d4f1f",
```

Comment:
As discussed, could you switched this id to the one we have in -extras? Thanks

Reply:
Done :+1:

Reply:
In manifest v2 there isn't a `guid` field anymore. I'm not sure if there's another field that we need to sync with extras too.